src/helpers/modifyDeck.js

```diff
@@ -0,0 +1,58 @@
+import resolveCardForLevel from './resolveCardForLevel'
+
+const setEqualLevels = deck => deck.map(card => ({ ...card, level: 1 }))
+
+export default (deck, modifier, equalLevels) => {
```

Comment:
```suggestion
export default (deck, modifier, equalsMode) => {
```

Maybe for consistency?